pom.xml

```diff
@@ -104,6 +104,45 @@
               </target>
             </configuration>
           </execution>
+          <execution>
+            <id>test</id>
+            <phase>test</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <property name="test_classpath" refid="maven.test.classpath"/>
```

Comment:
I suggest adding `<property name="build.compiler" value="extJavac"/>` here and checking whether `fork="true"` is still required.

Reply:
> I suggest adding `<property name="build.compiler" value="extJavac"/>` here and checking whether `fork="true"` is still required.

Thanks — tried adding that just now, and removing `fork="true"`, but I get the following error:

> An Ant BuildException has occured: java.lang.NoClassDefFoundError: jdk/internal/reflect/ConstructorAccessorImpl